docs/multi_process.md

```diff
@@ -0,0 +1,127 @@
+# Using JAX in multi-host and multi-process environments
+
+(test-ref)=
+## Introduction
+
+This guide explains how to use JAX in environments such as Cloud TPU pods where
+accelerators are spread across multiple CPU hosts or JAX processes. We’ll refer
+to these as “multi-process” environments.
+
+This guide specifically focuses on how to use collective communication
+operations (e.g. {func}`jax.lax.psum`) in multi-process settings, although other
+communication methods may be useful too depending on your use case (e.g. RPC,
+[mpi4jax](https://github.com/mpi4jax/mpi4jax)). If you’re not already familiar
+with JAX’s collective operations, we recommend starting with the
+{doc}`/jax-101/06-parallelism` notebook. Note that an important feature of
```

Comment:
Quick question/suggestion @skye : would the name of this page render on RTD when you use {doc}`/link`? If not, maybe it would help some users if there's a name of the notebook - _Parallel Evaluation in JAX_ (It's the 6th one but users may not see the numerical order when they come across JAX-101 at https://jax.readthedocs.io/en/latest/jax-101/index.html).

For example:

```suggestion
{doc}`/jax-101/06-parallelism` notebook (Parallel Evaluation in JAX). Note that an important feature of
```

Reply:
It does render the title! I forgot to post the link to the rendered version: https://jax--6796.org.readthedocs.build/en/6796/multi_process.html

(In general, you can see the rendered version of docs PRs by clicking on the "Details" of the "docs/readthedocs.org:jax" check)